InvenTree/company/migrations/0058_move_address_field_to_address_model.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 3.2.18 on 2023-05-02 20:41
+
+from django.db import migrations
+from company.models import Address
```

Comment:
Don't import models this way in a migration file - https://stackoverflow.com/questions/37765307/why-use-apps-get-model-when-creating-a-data-migration

Instead, use apps.get_model (as you have below for Company) e.g.

```python
Address = apps.get_model('company', 'Address')
```